Tests/MongoSwiftSyncTests/RetryableReadsTests.swift

```diff
@@ -54,8 +54,12 @@ final class RetryableReadsTests: MongoSwiftTestCase {
         self.continueAfterFailure = false
     }
 
-    func testRetryableReads() throws {
-        let tests = try retrieveSpecTestFiles(specName: "retryable-reads", asType: RetryableReadsTestFile.self)
+    func testRetryableReadsLegacy() throws {
+        let tests = try retrieveSpecTestFiles(
+            specName: "retryable-reads",
+            subdirectory: "legacy",
```

Comment:
these tests were moved into a `legacy` directory to distinguish them from newer, unified tests.